src/services/ipfs-service.js

```diff
@@ -31,40 +31,86 @@ class IpfsService {
     this.mapCache = new MapCache()
   }
 
-  async submitFile(jsonData) {
+  /**
+   * Convert an object to a JSON blob and submit it to IPFS.
+   *
+   * obj {string} - object to save as file
+   */
+  async saveObjAsFile(obj) {
+    let file
+    if (typeof Blob === 'undefined') {
+      file = new Buffer(JSON.stringify(obj))
```

Comment:
nit: looks like the node documentation says this method is deprecate. Buffer.from should be used. See https://nodejs.org/api/buffer.html#buffer_new_buffer_string_encoding